aspnetcore/fundamentals/minimal-apis/handle-errrors.md

```diff
@@ -156,3 +156,7 @@ app.Run();
 ```
 
 For more information on using `AddProblemDetails`, see [Problem Details](/aspnet/core/fundamentals/error-handling?view=aspnetcore-7.0&preserve-view=true#pds7)
+
+## Additional Resources
+
+* <xref:fundamentals/minimal-apis/security>
```

Comment:
Hmmmm...why do we want to link to the security doc from here?

Reply:
reverted